core/src/test/java/bio/terra/pearl/core/service/survey/SurveyTaskDispatcherTest.java

```diff
@@ -142,4 +141,28 @@ public void testAssignDoesntDuplicate(TestInfo testInfo) {
         assertThat(participantTasks, hasSize(2));
     }
 
+    @Test
+    @Transactional
```

Comment:
great test, let's add another one to confirm we don't assign surveys to non-subjects by default.